src/vm/dllimport.cpp

```diff
@@ -6194,18 +6204,16 @@ HINSTANCE NDirect::LoadLibraryModule(NDirectMethodDesc * pMD, LoadLibErrorTracke
     // even if it has one, or to leave off a prefix like "lib" even if it has one
     // (both of these are typically done to smooth over cross-platform differences). 
     // We try to dlopen with such variations on the original.
-    const WCHAR* const* prefixSuffixCombinations = nullptr;
-    int numberOfVariations = 0;
-    DetermineLibNameVariations(&prefixSuffixCombinations, &numberOfVariations, wszLibName, libNameIsRelativePath);
+    const WCHAR* prefixSuffixCombinations[4] = {};
```

Comment:
Suggest: Could we use a variable instead of 4? define maxCombinations as 4? and later we can just use this variable?

Reply:
I can do that the only reason I didn't is because the formats are statics at function scope rather than file scope. Having the constant disjoint from the function that performs the logic was something I wanted to avoid. Thoughts?

Reply:
Then just define a variable inside this function. or leave as it is.